corehq/apps/users/migrations/0037_add_edit_messaging_permission.py

```diff
@@ -0,0 +1,28 @@
+
+from django.db import migrations
+
+from dimagi.utils.chunked import chunked
+from corehq.apps.users.models_role import SQLPermission, UserRole
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_edit_migrations_permissions(apps, schema_editor):
+    permission, created = SQLPermission.objects.get_or_create(value='edit_messaging')
+    edit_data_permission = SQLPermission.objects.get(value='edit_data')
+    role_ids_with_edit_data = set(UserRole.objects.filter(rolepermission__permission_fk_id=edit_data_permission.id)
```

Comment:
how many records are we looking at updating here on prod?

Reply:
great question-- I ran the query and there are 3946 that will be updated

Reply:
That should update fairly quickly during deploy, yes?

Reply:
Oh that's a lot smaller than I would've guessed, nice.